### PR TITLE
[622<622-2] Add export section

### PR DESCRIPTION
**Pull request by vpavicic**
### Description

Add information about exporting

### CODE REVIEW

File: docs/export-data/overview.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+id: overview
+title: Export data
+sidebar_label: Export data
+slug: /export-data
+---
+
+Memgraph allows you to export all the data from the database, or query results.
```
Comment (vpavicic) [inline docs/export-data/overview.md:8]:
```suggestion
Memgraph allows you to export all the data from the database, or just results from an executed query.

```

**Comment by vpavicic** [comment]
I cannot add the top menu item because it is visible even in the 2.7 version of docs, not only next version... i can propagate this to all versions?

**Comment by vpavicic** [comment]
I thought about it and I think it's bad idea because we should track when each of these features was implemented... better to have it on the newest one only...

**Review by kgolubic — APPROVED** [review]
Everything looks OK

**Comment by vpavicic** [comment]
I wrote all of this in this comment KateLate :D 
https://github.com/memgraph/docs/pull/856#issuecomment-1527266833

1. So when I try to add the top menu item it doesn't pass the build so i just thought I'll add it once i merge this
2. About propagation - I'm not sure if we should propagate the large changes as well... like adding or removing new pages and stuff like that... 
3. I'll add Where to next... but... i where are you going next once you export stuff from memgraph? :D :D

**Comment by katarinasupe** [comment]
> I wrote all of this in this comment KateLate :D [#856 (comment)](https://github.com/memgraph/docs/pull/856#issuecomment-1527266833)
> 
> 1. So when I try to add the top menu item it doesn't pass the build so i just thought I'll add it once i merge this
That's fine.
> 2. About propagation - I'm not sure if we should propagate the large changes as well... like adding or removing new pages and stuff like that...
I think it is fine if we don't propagate this.
> 3. I'll add Where to next... but... i where are you going next once you export stuff from memgraph? :D :D
That is a good question. Maybe to backup and restore? 😂

**Comment by vpavicic** [comment]
> > 3. I'll add Where to next... but... i where are you going next once you export stuff from memgraph? :D :D
> 
> That is a good question. Maybe to backup and restore? 😂

Maybe even import again? 
Btw is creating a snapshot an import/export strategy?

**Comment by katarinasupe** [comment]
> Maybe even import again? Btw is creating a snapshot an import/export strategy? Or more like backup/restore thing

Hmm, yeah, import again is fine 😂 I like it!
That's a good question. I would somehow link to backup/restore on import overview. What do you think?

**Pull request merged**